content/cybersecurity/concepts/penetration-testing/penetration-testing.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: 'Penetration Testing'
+Description: 'Penetration Testing, also referred to as pen testing, involves breaking into a system to better identify cybersecurity weaknesses so that the security posture can be improved.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'paths/scan-systems-with-nmap'
+  - 'introduction-to-ethical-hacking'
+---
+
+**Penetration Testing**, also referred to as pen testing, involves breaking into a system to better identify cybersecurity weaknesses so that the security posture can be improved. It is typically conducted by [ethical hackers](https://www.codecademy.com/resources/docs/cybersecurity/hacking) who have an offensive security lens. 
```

Comment:
```suggestion
**Penetration testing**, also referred to as pen testing, involves attempting to break into a system to better identify cybersecurity weaknesses so that the security posture can be improved. It is typically conducted by [ethical hackers](https://www.codecademy.com/resources/docs/cybersecurity/hacking) who have an offensive security lens. 
```